justfile

```diff
@@ -31,3 +31,6 @@ add-transaction bundlePath to sig *params:
   mv {{bundlePath}} ${backupBundlePath}
   mv ${newBundlePath} {{bundlePath}}
   echo "Old bundle backed up to ${backupBundlePath}."
+
+clean:
+  forge clean
```

Comment:
This thin wrapper command obviously isn't necessary, but it felt more consistent with how we use `just` everywhere else.

Reply:
Agreed!